dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/GitHubUtil.java

```diff
@@ -110,18 +112,18 @@ public int createPullReq(GHRepository origRepo, String branch,
             log.warn("Handling error with pull request creation... {}", e.getMessage());
             JsonElement root = JsonParser.parseString(e.getMessage());
             JsonArray errorJson = root.getAsJsonObject().get("errors").getAsJsonArray();
-            String error = "";
-            if (errorJson.get(0).getAsJsonObject().has("message")) {
-                error = errorJson.get(0).getAsJsonObject().get("message").getAsString();
-            } else {
-                // This case usually happens when the PR head has invalid data. Deleting the forked repo resolves it.
-                error = PR_INVALID_CODE;
+            String error = PR_INVALID_CODE;
+            JsonObject errorMessage = errorJson.get(0).getAsJsonObject();
```

Comment:
Are there ever more than 1 error in this array? Any reason not to `foreach` on them?

Reply:
Not sure about that. This piece of code has been there for a while.